arroyo-worker/src/connectors/kinesis/source/mod.rs

```diff
@@ -370,6 +372,19 @@ impl<K: Data, T: Data + DeserializeOwned> KinesisSourceFunc<K, T> {
         )))
     }
 
+    async fn init_client(&mut self) {
+        let config = match self.aws_region {
+            Some(ref region) => {
+                aws_config::from_env()
+                    .region(Region::new(region.to_string()))
+                    .load()
+                    .await
+            }
+            None => load_from_env().await,
```

Comment:
I'm a bit confused why here we're using load_from_env(), but above we're using `from_env` — is this the same aside from the region override?

Reply:
`load_from_env()` is a very thin wrapper around `from_env()`, probably makes sense to just use `from_env()`. Here's `load_from_env()`:
```
/// Load a default configuration from the environment
///
/// Convenience wrapper equivalent to `aws_config::from_env().load().await`
pub async fn load_from_env() -> aws_types::SdkConfig {
    from_env().load().await
}